packages/amplify-provider-awscloudformation/src/root-stack-builder/root-stack-builder.ts

```diff
@@ -0,0 +1,252 @@
+import * as cdk from '@aws-cdk/core';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as iam from '@aws-cdk/aws-iam';
+import * as path from 'path';
+import { prepareApp } from '@aws-cdk/core/lib/private/prepare-app';
```

Comment:
We should not be depending on `private` part of CDK. This can change and we would not have any upgrade path when it does. Also I dont think they treat changes to private classes as breaking changes and could break CLI.